blockchain/fullblocktests/generate.go

```diff
@@ -2015,7 +2015,7 @@ func Generate(includeLargeReorg bool) (tests [][]TestInstance, err error) {
 	g.SetTip("brs3")
 	g.NextBlock("bmf25", outs[15], ticketOuts[15], func(b *wire.MsgBlock) {
 		spendOut := chaingen.MakeSpendableStakeOut(b, 0, 2)
-		ticketPrice := dcrutil.Amount(b.STransactions[5].TxOut[0].Value)
+		ticketPrice := dcrutil.Amount(g.CalcNextRequiredStakeDifficulty())
```

Comment:
This is not correct. It will calculate the ticket price for the next block, not the current one.